packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SVGComponents/VisXZoom/VisXZoom.js

```diff
@@ -42,32 +41,32 @@ const DEFAULT_HANDLER = () => true
  *       zoomInValue: 1.2,
  *       zoomOutValue: 0.8
  *     }}
+ *     onKeyDown={onKeyDown}
  *     setOnPan={setOnPan}
  *     setOnZoom={setOnZoom}
- *     zoomingComponent={SVGComponent}
- *     ...props
- *   />
+ *   >
+ *    {(zoomProps) => <SVGComponent {...zoomProps} />}
```

Comment:
The React docs recommend using a render function to make it explicit that a component injects props into its children.
https://react.dev/reference/react/cloneElement#alternatives